crates/ruff/src/message/diff.rs

```diff
@@ -47,8 +47,14 @@ impl Display for Diff<'_> {
 
         let diff = TextDiff::from_lines(self.source_code.source_text(), &output);
 
-        writeln!(f, "{}", "ℹ Suggested fix".blue())?;
-
+        let message = match self.fix.applicability() {
+            Applicability::Automatic => "Fix",
```

Comment:
Nit: What would you think of "Safe fix" (I'm undecided, Fix is short and to the point)?

Reply:
I'm unsure. It could also be "Automatic fix". I'm leaning towards brevity — it seems like an easy thing to change in the future if users are confused.